docs/content/preview/migrate/_index.md

```diff
@@ -10,12 +10,10 @@ type: indexpage
 
```

Comment:
Feel this page is now a waste - unless some other topic will be added? Otherwise, why not bump up the Voyager index page

Reply:
Yes , I feel there's no value to it. anymore. Okay will remove it.